lib/usdUfe/ufe/Utils.cpp

```diff
@@ -224,7 +239,11 @@ void setIsAttributeLockedFn(IsAttributeLockedFn fn)
 
 bool isAttributedLocked(const PXR_NS::UsdAttribute& attr, std::string* errMsg /*= nullptr*/)
 {
-    return gIsAttributeLockedFn ? gIsAttributeLockedFn(attr, errMsg) : false;
+    // If we have (optional) attribute is locked function, use it.
+    // Otherwise use the default one supplied by UsdUfe.
+    if (gIsAttributeLockedFn)
+        return gIsAttributeLockedFn(attr, errMsg);
+    return Editability::isAttributeLocked(attr, errMsg);
```

Comment:
Now that isAttributeLocked() was moved to UsdUfe, we have a default function to call.